src/lib/y2storage/storage_manager.rb

```diff
@@ -115,21 +129,27 @@ def increase_staging_revision
       @staging_revision += 1
     end
 
+    # All issues detected
+    #
+    # @return [Y2Issues::List<Issue>]
+    def issues
```

Comment:
I have doubts about this method name. Maybe is a bit too general. I foresee a future in which the BoootRequirementsChecker will also return a set of `Y2Issues`.  Should then `StorageManager.instance.issues` report also the boot-related issues found in the staging devicegraph? They will also be issues and they will also be associated to one of the devicegraphs managed by `StorageManager`.